ch04/ex-comments.md

```diff
@@ -94,3 +94,32 @@ There are some edge cases to take care of.
      <html>
        <body>
          'thing'
+
+
+
+Test case for handling text outside of comments
+
+    >>> test_parse("This is normal text --> following text should also be normal")
+     <html>
+       <body>
+         <this is="" normal="" text="" --="">
+           ' following text should also be normal'
+
```

Comment:
Let's just delete this test. It probably *is* how the browser behaves right now, but this is pretty weird behavior so maybe it's better not to test for it.